src/components/NcActions/NcActions.vue

```diff
@@ -1158,6 +1160,22 @@ export default {
 			type: Number,
 			default: 0,
 		},
+
+		/**
+		 * Specifies the button variant used for trigger and single actions buttons.
+		 *
+		 * Accepted values: primary, secondary, tertiary, tertiary-no-background, tertiary-on-primary, error, warning, success.
+		 * If left empty, the default button style will be applied.
+		 *
+		 * @since 8.23.0
+		 */
+		variant: {
+			type: String,
+			validator(value) {
+				return ['primary', 'secondary', 'tertiary', 'tertiary-no-background', 'tertiary-on-primary', 'error', 'warning', 'success'].indexOf(value) !== -1
```

Comment:
I'd prefer using `.includes(value)` here.
Can we also keep available variants in one place (as const, or util like `ValidateSlot`) to not duplicate it everywhere? In case it's not a limited list of only primary, secondary, tertiary

Reply:
> I'd prefer using .includes(value) here.

yes me too :) 

>  Can we also keep available variants in one place (as const, or util like ValidateSlot) to not duplicate it everywhere? In case it's not a limited list of only primary, secondary, tertiary

I will do so on v9 with #6033 as then we can share the types directly.